package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "ot-hapi-queryparams",
-  "version": "0.2.2",
+  "version": "1.0.0",
   "description": "Hapi.js plugin for converting query params",
   "main": "index.js",
   "directories": {
     "test": "test"
   },
   "dependencies": {
-    "joi": "^6.10.1"
+    "joi": "^14.0.0",
+    "ot-schemas": "git+https://github.com/opentable/ot-schemas.git"
```

Comment:
this is a public facing repository (it's available to anyone on the internet). 
The ot-schemas repo is a private repository. 
This won't work

Reply:
this is an open source project